monix-reactive/shared/src/main/scala/monix/reactive/Consumer.scala

```diff
@@ -227,6 +229,17 @@ object Consumer {
   def raiseError[In, R](ex: Throwable): Consumer.Sync[In, R] =
     new RaiseErrorConsumer(ex)
 
+  /** A consumer that will return a collection of all the streamed
+   * values as a Scala `List`.
+   *
+   * Note that this method is functionally the same as the [[Observable.toListL]]
+   *
+   * WARNING: For infinite streams the process will eventually
+   * blow up with an out of memory error.
+   */
+  def toListL[A]: Consumer[A, List[A]] =
```

Comment:
`L` is only used in `Observable` to differentiate between methods returning `Task` and `Observable`, I would leave just `toList` in `Consumer`. Sorry for nitpick :D

Reply:
No worries, I have changed that already :)